ep_testing/config.py

```diff
@@ -27,6 +27,9 @@ class OS:
     'win64': {
         'os': OS.Windows, 'bitness': 'x64', 'asset_pattern': 'Windows-x86_64.zip', 'os_version': '10'
     },
+    'win64-2022server': {
+        'os': OS.Windows, 'bitness': 'x64', 'asset_pattern': 'Windows-x86_64.zip', 'os_version': '2022'
+    },
```

Comment:
Implement a new config option that sets the os version according to the windows server version.  This will be the indicator that we should use the newer visual studio version.